care/emr/api/viewsets/payment_reconciliation.py

```
@@ -105,6 +105,14 @@ def authorize_create(self, instance):
             if invoice.facility != facility:
                 raise ValidationError("Invoice is not associated with the facility")
 
+    def authorize_update(self, request_obj, model_instance):
```

Ruff (0.13.1)

Unused method argument: `request_obj`

(ARG002)